Makefile

```diff
@@ -176,6 +176,16 @@ go.sum: go.mod
 ###                              Documentation                              ###
 ###############################################################################
 
+update-swagger-docs: statik
+	$(BINDIR)/statik -src=client/docs/swagger-ui -dest=client/docs -f -m
+	@if [ -n "$(git status --porcelain)" ]; then \
+        echo "\033[91mSwagger docs are out of sync!!!\033[0m";\
+        exit 1;\
+    else \
+        echo "\033[92mSwagger docs are in sync\033[0m";\
+    fi
+.PHONY: update-swagger-docs
+
```

Comment:
I am not sure if this is removed intentionally, I couldn't find an handy command to generate swagger docs

Reply:
@alexanderbez do you have any context on this?

Reply:
The `statik` dependency was removed long ago. Not sure why we still have this. The goal of the target was to ensure the swagger docs were up to date.

Reply:
Okay, I will remove it and raise a PR